src/components/GameView/MoveChoiceOverlay.vue

```diff
@@ -97,7 +97,7 @@ export default {
 			}
 			// Re-usable move-objects
 			// Points
-			const pointsDescription =  cardRank === 1 ? 'Gain 1 point' : `Gain ${cardRank} points`;
+			const pointsDescription =  `Gain ${cardRank} point` + (cardRank===1?null:'s');
```

Comment:
Should we use an empty string instead of null for the singular case?

Reply:
Yep, good point